source/gui/settingsDialogs.py

```diff
@@ -2008,6 +2026,7 @@ def onSave(self):
 		config.conf["keyboard"]["speakCommandKeys"] = self.commandKeysCheckBox.IsChecked()
 		config.conf["keyboard"]["alertForSpellingErrors"] = self.alertForSpellingErrorsCheckBox.IsChecked()
 		config.conf["keyboard"]["handleInjectedKeys"] = self.handleInjectedKeysCheckBox.IsChecked()
+		config.conf["keyboard"]["maxRepeatedKeyPressDelay"] = self.maxRepeatedDelayEdit.GetValue()
```

Comment:
There is no risk to have a config with invalid value because:
- `self.maxRepeatedDelayEdit` already ensure data validation in the expected range (see my answer to your first comment above)
- in any case, the config spec also ensures that no invalid value will be saved in the config: trying to do so will raise an error.
For the PRs in this project, could you remember to look at the config spec to check that we do not put unexpected values in the config? Thanks.